README.md

```diff
@@ -1,2 +1,5 @@
 # JDBC Connectors for Apache Hive and Presto on Cloud Dataproc
 
+* To run unit test, use `./gradlew test --tests="com.google.cloud.dataproc.jdbc.{test name}"`
```

Comment:
+1 to adding this information to the readme. Let's add a section header like "## Running tests".

Also, does it run all unit tests if you just run `./gradlew test` ? If so, that might be a bit shorter to document.